tests/functional/test_globals.py

```diff
@@ -0,0 +1,21 @@
+import os
+import unittest
+
+from awscli.testutils import create_clidriver
+from awscli.clidocs import EXAMPLES_DIR, GlobalOptionsDocumenter
+
+
+class TestGlobalOptionsDocumenter(unittest.TestCase):
+    def setUp(self):
+        self.driver = create_clidriver()
+        self.globals = GlobalOptionsDocumenter(self.driver)
+
+    def test_global_options(self):
```

Comment:
For the test case name, let's make it a bit more explicit about what property we are asserting to be more explicit what the purpose of the test is. For example, let's name this to something like: `test_doc_global_options_match_saved_content()`. I'm up for suggestions, but mainly want to make sure we are capturing the purpose of this test which is what we generate is the same as what is stored on disk/in the repository.